packages/client/components/ActivityLibrary/ActivityLibrary.tsx

```diff
@@ -1,12 +1,33 @@
-import React from 'react'
 import graphql from 'babel-plugin-relay/macro'
+import React from 'react'
 import {PreloadedQuery, usePreloadedQuery} from 'react-relay'
-import {ActivityLibraryQuery} from '~/__generated__/ActivityLibraryQuery.graphql'
 import {Redirect} from 'react-router'
+import {ActivityLibraryQuery} from '~/__generated__/ActivityLibraryQuery.graphql'
+import ActivityLibrarySideBar from './ActivityLibrarySideBar'
+import ActivityLibraryCard from './ActivityLibraryCard'
+
+graphql`
+  fragment ActivityLibrary_template on MeetingTemplate {
+    id
+    teamId
+    team {
+      name
+    }
+    name
+    type
+  }
+`
 
 const query = graphql`
   query ActivityLibraryQuery {
     viewer {
+      availableTemplates(first: 100) @connection(key: "ActivityLibrary_availableTemplates") {
```

Comment:
+1 github lost my comment, or I somehow deleted it. it's a little bit of overkill to have a connection  here, as we won't have too many, no harm though

Reply:
Hmm I don't disagree, but since the existing `organizationTemplates` and `publicTemplates` fields on the meeting settings objects are connections, I have a weak preference to either a) make this a connection, too, or b) make this an array and make those fields arrays.